### PR TITLE
updated for PM review

### PR DESCRIPTION
**Pull request by mcherem**
PM review.

### CODE REVIEW

**Comment by mcherem** [comment]
Merging Amit's changes from the v2.0 branch to be in line with the changes I made in this PR.

File: modules/ROOT/pages/B2B-overview.adoc [hunk 1]
```diff
@@ -12,41 +13,47 @@ Before you configure your message flows to send and receive messages with your p
 You’ll also need an understanding of how your backend systems work.
 Use this checklist to help organize the information you need to gather before you configure your host, partner, and message flows.
 
-* Information you'll need to have on hand as the host organization:
+Information you'll need to have on hand as the host organization:
+
+
+* Your organizations identifiers (such as the AS2 Identifier, X12 ISA, X12 GS, and so on)
```
Comment (wwarner-salesforce) [inline modules/ROOT/pages/B2B-overview.adoc]:
```suggestion
* Your organization's identifiers (such as the AS2 Identifier, X12 ISA, X12 GS, and so on)
```

File: modules/ROOT/pages/B2B-overview.adoc [hunk 2]
```diff
@@ -12,41 +13,47 @@ Before you configure your message flows to send and receive messages with your p
...
-** Partner name (usually the name of the company)
 
-** Your trading partners identifiers (such as the AS2 Identifier, X12 ISA, X12 GS, and so on)
+Information needed for each trading partner:
 
-** Trading Partner Endpoints - where you send transmissions to your partners, including acknowledgement endpoints
+* Partner name (usually the name of the company)
 
-** Electronic Data Interchange (EDI) message format preferences for your partner (such as X12),
+* Your trading partners identifiers (such as the AS2 Identifier, X12 ISA, X12 GS, and so on)
```
Comment (wwarner-salesforce) [inline modules/ROOT/pages/B2B-overview.adoc]:
```suggestion
* Your trading partner's identifiers (such as the AS2 Identifier, X12 ISA, X12 GS, and so on)
```

File: modules/ROOT/pages/B2B-overview.adoc [hunk 3]
```diff
@@ -12,41 +13,47 @@ Before you configure your message flows to send and receive messages with your p
...
+
+* Electronic Data Interchange (EDI) message format preferences for your partner (such as X12),
 and any associated validations that require an acknowledgment (such as TA1, 997, 999, and so on)
 
-** Message types to use. For example, a purchase order in X12 is an 850 message.
+* Message types to use. For example, a purchase order in X12 is an 850 message.
 You also need to know the X12 version to use in your transactions (such as 004010, 005010, and so on).
 
-** Your partners' digital certificates (such as X.509)
+* Your partners' digital certificates (such as X.509)
```
Comment (wwarner-salesforce) [inline modules/ROOT/pages/B2B-overview.adoc]:
```suggestion
* Your partner's digital certificates (such as X.509)
```

File: modules/ROOT/pages/configure-message-flows.adoc [hunk 4]
```diff
@@ -3,21 +3,23 @@ ifndef::env-site,env-github[]
...
 You'll need the following for the Source:
 
-* The partner and associated identifiers.
-* The endpoint where you receive the transmission.
-* The message type to process for this message flow.
-The Dataweave map that transforms the message contents into a form your systems can use.
+* The partner and associated identifiers
+* The endpoint where you receive the transmission
+* The message type to process for this message flow
+The Dataweave map that transforms the message contents into a form your systems can use
```
Comment (wwarner-salesforce) [inline modules/ROOT/pages/configure-message-flows.adoc]:
```suggestion
* The Dataweave map that transforms the message contents into a form your systems can use
```

File: modules/ROOT/pages/index.adoc [hunk 5]
```diff
@@ -13,8 +13,9 @@ They are typically transmitted over transports (such as HTTP(s), SFTP, and excha
 All of this is integrated into the Mule runtime.
 
 Use Anypoint Partner Manager to set up, manage, and monitor the flow of messages between you and your business partners.
-Create message flows, the paths along which a message travels from its source, through a transformation map, to its target.
-Monitor each transmission, the transfer of a payload and the validation of the messages it contains. The ability to track and report on this drives greater operational efficiency while managing B2B transactions.
+Create message flows (the paths along which a message travels from its source) through a transformation map, to its target.
```
Comment (wwarner-salesforce) [inline modules/ROOT/pages/index.adoc]:
```suggestion
Create message flows. A message flow is the path along which a message travels from its source, through a transformation map, to its target.
```

Comment (mcherem) [reply modules/ROOT/pages/index.adoc]: This was edited by Kim and if Amit didn't change it, I"m inclined to leave it.

Comment (wwarner-salesforce) [reply modules/ROOT/pages/index.adoc]: The original was "Create message flows, the paths along which a message travels from its source, through a transformation map, to its target." That sounds to me like the definition of a message flow is "the path along which a message travels from its source, through a transformation map, to its target". I think "Create message flows (the paths along which a message travels from its source) through a transformation map, to its target." garbles the definition incorrectly, so I'd recommend rewording this.

**Review by wwarner-salesforce — APPROVED** [review]

**Pull request merged**